core/whistle_apps-1.0.0/src/whapps_util.erl

```diff
@@ -288,6 +291,47 @@ get_account_by_realm(RawRealm) ->
     Realm = wh_util:to_lower_binary(RawRealm),
     get_accounts_by(Realm, ?ACCT_BY_REALM_CACHE(Realm), ?AGG_LIST_BY_REALM).
 
+-spec get_account_by_ip(ne_binary()) ->
+                               {'ok', wh_proplist()} |
+                               {'error', 'not_found'}.
+get_account_by_ip(IP) ->
```

Comment:
@jamesaimonetti  I would argue that get_account_by_ip should return the matching document then the modules convert to CCVs.  Or maybe call it get_ccvs_by_ip?